### PR TITLE
{bio}[goolf/1.7.20] LUMPY v0.2.13 + gawk v4.0.2

### PR DESCRIPTION
**Pull request by pescobar**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by pescobar** [comment]
depends on https://github.com/hpcugent/easybuild-easyconfigs/pull/4435 and https://github.com/hpcugent/easybuild-easyconfigs/pull/4442

**Comment by boegel** [comment]
@pescobar still missing one?

```
ERROR: Irresolvable dependencies encountered: SAMtools/1.3.1-goolf-1.7.20
```

**Comment by boegel** [comment]
@pescobar ping on missing easyconfig?

**Comment by boegel** [comment]
@pescobar Ping again (same one missing as in #5490...)

**Comment by boegel** [comment]
(close/open to retrigger Travis)

File: easybuild/easyconfigs/l/LUMPY/LUMPY-0.2.13-goolf-1.7.20.eb [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+version = '0.2.13'
+
+homepage = 'https://github.com/arq5x/lumpy-sv'
+description = 'lumpy: a general probabilistic framework for structural variant discovery'
+
+toolchain = {'name': 'goolf', 'version': '1.7.20'}
+
+# no source_urls because the tarball has to be manually generated doing 
+# git clone --recursive https://github.com/arq5x/lumpy-sv.git
+sources = ['%(name)s-%(version)s.tar.gz']
```
Comment (boegel) [inline easybuild/easyconfigs/l/LUMPY/LUMPY-0.2.13-goolf-1.7.20.eb]: @pescobar Missing `SHA256` checksum?

Also, seems like there are proper tarballs now, see https://github.com/arq5x/lumpy-sv/releases/download/0.2.13/lumpy-sv-v0.2.13.tar.gz ?

File: easybuild/easyconfigs/l/LUMPY/LUMPY-0.2.13-goolf-1.7.20.eb [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+dependencies = [
+    ('Python', '2.7.11'),
+    ('zlib', '1.2.8'),
+    ('samblaster', '0.1.24'),
+    ('SAMtools', '1.3.1'),
+    ('Sambamba', '0.6.6', '', True),
+]
+
+osdependencies = [('gawk')]
```
Comment (boegel) [inline easybuild/easyconfigs/l/LUMPY/LUMPY-0.2.13-goolf-1.7.20.eb]: Why not as a proper dep through EasyBuild? We have (old) easyconfigs for `gawk`...

File: easybuild/easyconfigs/l/LUMPY/LUMPY-0.2.13-goolf-1.7.20.eb [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+skipsteps = ['configure', 'install']
+
+buildininstalldir = True
```
Comment (boegel) [inline easybuild/easyconfigs/l/LUMPY/LUMPY-0.2.13-goolf-1.7.20.eb]: maybe `MakeCp` is better as easyblock?

File: easybuild/easyconfigs/l/LUMPY/LUMPY-0.2.13-goolf-1.7.20.eb (hunk 1 above)
Comment (pescobar) [reply easybuild/easyconfigs/l/LUMPY/LUMPY-0.2.13-goolf-1.7.20.eb]: @boegel tarballs are provided but don't include all the required sources
https://github.com/arq5x/lumpy-sv/blob/master/.gitmodules

File: easybuild/easyconfigs/l/LUMPY/LUMPY-0.2.13-goolf-1.7.20.eb [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+name = 'LUMPY'
+version = '0.2.13'
+
+homepage = 'https://github.com/arq5x/lumpy-sv'
+description = 'lumpy: a general probabilistic framework for structural variant discovery'
+
+toolchain = {'name': 'goolf', 'version': '1.7.20'}
+
+# no source_urls because the tarball has to be manually generated doing
+# git clone --recursive https://github.com/arq5x/lumpy-sv.git
```
Comment (boegel) [inline easybuild/easyconfigs/l/LUMPY/LUMPY-0.2.13-goolf-1.7.20.eb]: @pescobar These instructions are incomplete; to get exactly `0.2.13`, you need to do a `git reset --hard f194e6f` too:

```
git clone --recursive https://github.com/arq5x/lumpy-sv.git
cd lumpy-sv
git reset --hard f194e6f  # specific to v0.2.13
cd ..
mv lumpy-sv lumpy-sv-0.2.13
tar cfvz LUMPY-0.2.13.tar.gz lumpy-sv-0.2.13
```

Comment (pescobar) [reply easybuild/easyconfigs/l/LUMPY/LUMPY-0.2.13-goolf-1.7.20.eb]: I have updated the instructions to create the tarball. Thanks!

**Comment by boegel** [comment]
@pescobar Sanity check fails for me, no `bin/lumpy_filter` found?

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in this PR)
node2106.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/9daada08003df6ea8a082954e7a21c0e for a full test report.

**Comment by pescobar** [comment]
Test report by @pescobar
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
login20.cluster.bc2.ch - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 v2 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/8bbf08305fe8df398b2314b47674bd14 for a full test report.

**Comment by pescobar** [comment]
@boegel it seems that in my cluster I installed latest code from master branch (I didn't do the `git reset --hard f194e6f`) and this includes the `lumpy_filter` binary which is not included in version 2.3

I have removed this binary from the sanity check

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2098.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/f7dea96414daa238c925cf9aac85d40d for a full test report.

**Comment by boegel** [comment]
Going in, thanks @pescobar!

**Pull request merged**